### ISSUE TITLE
Unhandled NotFoundException thrown on client.get()

### ISSUE THREAD

**Issue by phy** [driftx/Telephus#1]
There seems to be two NotFoundException's thrown when a get() for a nonexistent item is requested.  The first NotFoundException is catchable but the 2nd isn't.  e.g. running

```
try:
    r = yield self.client.get(key=key, columnPath=columnfamily,
        column=column)
except:
    print "this catches the first exception, but not the 2nd"
```

will print "this catches the first exception, but not the 2nd" and a Failure: telephus.cassandra.ttypes.NotFoundException: NotFoundException()

I haven't looked deeper to find out why but I thought I would give a heads up.

**Comment by grep0** [comment]
I've got the same problem and fixed in my fork: http://github.com/grep0/Telephus/commit/012cdc21e0ccb4c3caa68db33a9fd58d0b149d7d

**Comment by driftx** [comment]
Fixed in 0.6.2
